### ISSUE TITLE
[Next.js] TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by nandorojo** [nandorojo/moti#10]
Using `v0.0.5`, I'm having trouble running `moti` in Next.js.

```jsx
import React from 'react'
import { View } from 'moti'

export default function Test() {
  return <View />
}
```

This code produces the following error:

<img width="917" alt="Screen Shot 2021-02-01 at 10 50 59 PM" src="https://user-images.githubusercontent.com/13172299/106549913-fe306080-64df-11eb-9d06-b53539196364.png">

My best guess is some use of `Object.keys` in `useAnimatedStyle` is breaking. Not sure.

**Comment by nandorojo** [comment]
@cmaycumber would it be hard to test up a nextjs example like we have for Dripsy here?

**Comment by nandorojo** [comment]
I tried everything from https://github.com/software-mansion/react-native-reanimated/issues/1090 but no luck.

**Comment by cmaycumber** [comment]
I can put one together! It should be the exact same as the Dripsy one

**Comment by nandorojo** [comment]
Perfect. I'm using the `next` branch rn by the way

**Comment by cmaycumber** [comment]
> Perfect. I'm using the `next` branch rn by the way.
> 
> Thanks for the quick help here!

Should be able to get this in today!

**Comment by nandorojo** [comment]
@cmaycumber Awesome!

**Comment by nandorojo** [comment]
I think I figured this out, mentioned it here: https://github.com/software-mansion/react-native-reanimated/issues/1090#issuecomment-774155665

Pushing example to `next` branch shortly.

**Comment by cmaycumber** [comment]
Sweet sorry this took so long, got busier than I expected.

**Comment by nandorojo** [comment]
No worries!

**Comment by nandorojo** [comment]
Fixed in `0.0.7`!

`yarn add moti`
